### PR TITLE
Added @thewchan as maintainer; updated licenses and files, added pyth…

### PR DESCRIPTION
**Pull request by thewchan**
…on min requirement, bumped build version

Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

Hi I would like to add myself as a maintainer for the purpose of updating the package, as it is a dependency for another package I'm trying to deploy on Conda-Forge

### CODE REVIEW

**Comment by thewchan** [comment]
There are three separate licenses, I'm not super sure how to add those properly in the `meta.yml` , right now I've entered them as a comma-delimited string @conda-forge/core

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,34 +1,40 @@
...
+  commands:
+    - pip check
+  requires:
+    - pip
 
 about:
   home: http://pyphen.org/
-  license: GPL 2.0+/LGPL 2.1+/MPL 1.1
-  license_file: COPYING
+  license: GPL-2.0-or-later, LGPL-2.1-or-later, MPL-1.1
```
Comment (isuruf) [inline recipe/meta.yaml]:
```suggestion
  license: GPL-2.0-or-later AND LGPL-2.1-or-later AND MPL-1.1
```
If it's an OR, you can replace AND with OR above

**Comment by thewchan** [comment]
@conda-forge/core

There is some error when building because of legacy python2 and old pip version code, any suggestions how to fix that?

File: recipe/meta.yaml (hunk 1 above)
Comment (thewchan) [reply recipe/meta.yaml]: Gotcha, I will make that change. Thanks! @isuruf

**Comment by jakirkham** [comment]
Need to re-render (please see command in PR template)

**Comment by thewchan** [comment]
@conda-forge-admin, please rerender

**Comment by jakirkham** [comment]
Might consider adding the same email used for the commits to your GitHub account. That way GitHub will associate these commits with your username 🙂

**Comment by thewchan** [comment]
@jakirkham I swear I've had that set up but I don't know why the config disappeared 🤷 

@isuruf All checks have passed now, thanks!

**Comment by jakirkham** [comment]
Yeah sometimes if I use a different machine, I find that a different email is setup than I would have assumed (or default email is setup that doesn't actually exist). Anyways can appreciate this is not always easy to setup right 🙂

**Comment by thewchan** [comment]
Thanks @jakirkham! Do you have a good suggestion of a workflow for adding branches for past versions? The reason I ask is I think I need to add them for dependencies for other packages I'm trying to deploy

**Comment by jakirkham** [comment]
Are we maintaining this long term or is this a one off thing?

**Comment by thewchan** [comment]
Would long term be a problem? It eats up too many resource? Can a past version exist only for a bit for another package as a dependency only to be discarded later?

**Comment by jakirkham** [comment]
Not at all. It just changes what we do. In either case would submit that as a 2nd PR after this. For long term, we would just add a new branch for that version based off `master`

**Comment by thewchan** [comment]
I think having branches makes sense; but I'm a little unclear on how to get the past versions built. Do I just specify a different version number on the `meta.yml` file? (On my own fork and a dedicated branch of course)

**Comment by jakirkham** [comment]
Yep it is really that simple 🙂

Can even PR to `master` to start. Then we can add a new branch and retarget it

**Comment by thewchan** [comment]
Nice, thanks so much for the help! I'll go try this

**Comment by thewchan** [comment]
@isuruf would you be able to review this? Thanks.

**Comment by rmax** [comment]
Looks good to me!

**Review by rmax — APPROVED** [review]

**Comment by jakirkham** [comment]
Thanks @rmax! 😄

**Pull request merged**